src/Common/RJSF/templates/ArrayFieldItemTemplate.tsx

```diff
@@ -32,16 +32,18 @@ export const ArrayFieldItemTemplate = ({
     const { RemoveButton } = registry.templates.ButtonTemplates
 
     return (
-        <div className="display-grid rjsf-form-template__array-field-item flex-align-center dc__gap-8 mb-12">
+        <div className="dc__position-rel display-grid rjsf-form-template__array-field-item flex-align-center mb-12">
```

Comment:
Should we get rid of the gap here? What if the children has multiple elements?

Reply:
The gap can be safely removed since div only contains the container for the (intput + label) & the cross button